kokoro/release/python/macos/config.sh

```diff
@@ -25,12 +25,13 @@ function pre_build {
     # Runs in the root directory of this repository.
     pushd protobuf
 
-    # Build protoc
-    ./autogen.sh
-    ./configure
-
-    CXXFLAGS="-std=c++14 -fPIC -g -O2" ./configure
-    make -j8
+    # Build protoc and protobuf libraries
+    use_bazel.sh 5.1.1
+    bazel build //:protoc
+    export PROTOC=$PWD/bazel-bin/protoc
+    mkdir src/.libs
+    ln -s $PWD/bazel-bin/libprotobuf.a src/.libs/libprotobuf.a
+    ln -s $PWD/bazel-bin/libprotobuf_lite.a src/.libs/libprotobuf-lite.a
```

Comment:
On a side note, there is a flag that you can pass to setup.py to specify the libs directory:

https://github.com/pypa/setuptools/blob/main/setuptools/_distutils/command/build_ext.py#L76

However, that would have to be passed to (I think) the bdist_wheel command, not the build_py command.

Either way, though, this should work.

Reply:
Thanks for the pointer! I think I will leave it how it is now though, only because these lines are likely going to be deleted very soon anyway when Python-UPB is released.